examples/channel_inversion/Makefile

```diff
@@ -0,0 +1,18 @@
+all: invert plot
+
+CONTROLS	= Bathymetry Manning InitialElev {Bathymetry,Manning} {Manning,InitialElev} {InitialElev,Bathymetry} {Bathymetry,Manning,InitialElev}
```

Comment:
I'm not sure if running all of the combinations makes sense, at least if the `Makefile` intended to demonstrate the capabilities to the user. I've really only tested the individual ones.